### PR TITLE
Fixes/stripe improvements

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

- Cleans up remaining env vars in stripe App
- Adds proper schema for Stripe metadata
- Allows to return to where you were if installing App from OmniAppInstall
- Adds missing "paid" badge to bookings

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] Install Stripe from the event type edit page suggestions
- [ ] You should go back to that page with Stripe installed

### CODE REVIEW

File: packages/app-store/stripepayment/lib/client/createPaymentLink.ts [hunk 1]
```diff
@@ -11,7 +13,7 @@ export function createPaymentLink(opts: {
 }): string {
   const { paymentUid, name, email, date, absolute = true } = opts;
   let link = "";
-  if (absolute) link = process.env.NEXT_PUBLIC_WEBSITE_URL!;
+  if (absolute) link = WEBSITE_URL;
```
Comment (rene-mueller) [inline packages/app-store/stripepayment/lib/client/createPaymentLink.ts:16]: is there a `!` missing?

File: apps/web/components/booking/BookingListItem.tsx [hunk 2]
```diff
@@ -280,6 +280,11 @@ function BookingListItem(booking: BookingItemProps) {
                 {t("paid")}
               </Badge>
             )}
+            {booking.paid && (
+              <Badge className="ltr:mr-2 rtl:ml-2" variant="green">
+                {t("paid")}
+              </Badge>
+            )}
```
Comment (alannnc) [inline apps/web/components/booking/BookingListItem.tsx:287]: This is the same code as above?

Comment (alannnc) [inline apps/web/components/booking/BookingListItem.tsx:287]:
```suggestion

```

Comment (zomars) [reply apps/web/components/booking/BookingListItem.tsx:287]: Huh. It merge conflict it seems

**Review by alannnc — APPROVED** [review]

**Pull request merged**